### ISSUE TITLE
Infinity timestamp format preventing data grid editing

### ISSUE THREAD

**Issue by pdesuza** [dbeaver/dbeaver#12639]
#### System information: 
- MacOS Catalina v10.15.7
- DBeaver v21.0.5.202105160657

#### Connection specification:
- Postgres 11.11
- PostgresSQL JDBC Driver
- Do you use tunnels or proxies (SSH, SOCKS, etc)? No

#### Describe the problem you're observing:
Trying to edit column in data grid but complains about +Infinity formatting no matter what field I edit

```
Error synchronizing data with database

Reason:
SQL Error [22007]: ERROR: invalid input syntax for type timestamp: "+Infinity"
```
The column it's complaining about is defined as `TIMESTAMP DEFAULT ('infinity'::timestamp at time zone 'utc')`
I used to be able to edit and save cells no problem

![Screen Shot 2021-05-27 at 6 52 56 PM](https://user-images.githubusercontent.com/20630834/119918737-e4e37200-bf1d-11eb-8f8d-8b4ae330d8ec.png)
![Screen Shot 2021-05-27 at 6 51 47 PM](https://user-images.githubusercontent.com/20630834/119918743-e6ad3580-bf1d-11eb-9c21-8bfa79d452c1.png)

#### Steps to reproduce, if exist:
1. Select * from any table that has column defined like that
2. Edit any column in the resulting data grid

#### Include any warning/errors/backtraces from the logs
```
eclipse.buildId=unknown
java.version=11.0.8
java.vendor=AdoptOpenJDK
BootLoader constants: OS=macosx, ARCH=x86_64, WS=cocoa, NL=en_US
Framework arguments:  -keyring /Users/pdesuza/.eclipse_keyring
Command-line arguments:  -os macosx -ws cocoa -arch x86_64 -keyring /Users/pdesuza/.eclipse_keyring

org.jkiss.dbeaver.model
Error
Thu May 27 18:56:16 PDT 2021
ERROR: invalid input syntax for type timestamp: "+Infinity"

org.postgresql.util.PSQLException: ERROR: invalid input syntax for type timestamp: "+Infinity"
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2440)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2183)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:308)
	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:441)
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:365)
	at org.postgresql.jdbc.PgPreparedStatement.executeWithFlags(PgPreparedStatement.java:143)
	at org.postgresql.jdbc.PgPreparedStatement.execute(PgPreparedStatement.java:132)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCPreparedStatementImpl.execute(JDBCPreparedStatementImpl.java:261)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCPreparedStatementImpl.executeStatement(JDBCPreparedStatementImpl.java:205)
	at org.jkiss.dbeaver.model.impl.data.ExecuteBatchImpl.executeStatement(ExecuteBatchImpl.java:304)
	at org.jkiss.dbeaver.model.impl.data.ExecuteBatchImpl.processBatch(ExecuteBatchImpl.java:172)
	at org.jkiss.dbeaver.model.impl.data.ExecuteBatchImpl.execute(ExecuteBatchImpl.java:81)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetPersister$DataUpdaterJob.executeStatements(ResultSetPersister.java:811)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetPersister$DataUpdaterJob.lambda$1(ResultSetPersister.java:697)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:169)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetPersister$DataUpdaterJob.executeStatements(ResultSetPersister.java:696)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetPersister$DataUpdaterJob.run(ResultSetPersister.java:634)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetPersister.execute(ResultSetPersister.java:450)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetPersister.applyChanges(ResultSetPersister.java:174)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetViewer.saveChanges(ResultSetViewer.java:3878)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetViewer.applyChanges(ResultSetViewer.java:3851)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetViewer.doSave(ResultSetViewer.java:2200)
	at org.jkiss.dbeaver.ui.editors.sql.SQLEditor$SaveJob.run(SQLEditor.java:3906)
	at org.jkiss.dbeaver.model.runtime.AbstractJob.run(AbstractJob.java:105)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

```

**Comment by ShadelessFox** [comment]
Thanks for the report.

**Comment by DariaMarkaryan** [comment]
Positive representation converts to negative on saving
![2021-06-02_17h13_02](https://user-images.githubusercontent.com/43252954/120496171-059c4380-c3c6-11eb-8243-44d972807bf1.gif)

**Comment by DariaMarkaryan** [comment]
Verified

**Comment by pdesuza** [comment]
Hi Team,

Thanks for looking into and working on this. Any idea of when the fix will go live?

**Comment by ShadelessFox** [comment]
Hello,

This fix will go live when 21.1.1 is released. It usually happens on Mondays, so somewhere in the span of 24 hours :)

**Comment by pdesuza** [comment]
You're awesome! Thank you so much for your efforts.
